jpsreport/methods/ConfigData_D.h

```diff
@@ -47,5 +50,27 @@ struct ConfigData_DIJ {
     //general parameters
     bool isOneDimensional = false;
     bool useBlindPoints   = true;
+
+    std::string densityType  = "Voronoi";
+    std::string velocityType = "Voronoi";
+
+    //parameter for velocity calculation function
+    /** default:
+    Calculates the velocity of pedestrians based on their voronoi cell and their instantaneous velocity.
+    input: intersecting polygons (with MA), instantaneous velocities and measurement area.
+    output: mean velocity in the MA
+    **/
```

Comment:
Can you make this as doxygen comment and fuse merge both?

Reply:
Done